src/libwalletqt/Wallet.cpp

```diff
@@ -215,6 +215,16 @@ PendingTransaction *Wallet::createTransaction(const QString &dst_addr, const QSt
     return result;
 }
 
+PendingTransaction *Wallet::createTransactionAll(const QString &dst_addr, const QString &payment_id,
+                                                 quint32 mixin_count, PendingTransaction::Priority priority)
+{
+    Bitmonero::PendingTransaction * ptImpl = m_walletImpl->createTransaction(
+                dst_addr.toStdString(), payment_id.toStdString(), boost::optional<uint64_t>(), mixin_count,
```

Comment:
Yes, I just wanted to keep interface without any additional dependencies until this really needed. src/wallet/api - yes it uses boost, but not in public interface.